apps/arweave/src/arweave.app.src

```diff
@@ -0,0 +1,19 @@
+{application, arweave, [
+	{description, "Arweave"},
+	{vsn, "1.9.0.0"},
+	{mod, {arweave_app, []}},
+	{registered, []},
+	{applications, [
+		kernel,
+		stdlib,
+		os_mon,
```

Comment:
We need to update `application:start(sasl), application:start(os_mon)` in `ar_storage.erl`.

Reply:
Updated.